apps/omg_eth/lib/omg_eth/ethereum_height_monitor/alarm_handler.ex

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Eth.EthereumHeightMonitor.AlarmHandler do
+  @moduledoc """
+  Listens for :ethereum_connection_error and :ethereum_stalled_sync alarms and reflect
+  the alarm's state back to the monitor.
+  """
+  use GenServer
```

Comment:
as said in slack, this isn't really a genserver, the state and the execution is from :gen_event

if you remove this is enough, just so there's no further confusion from possible readers.

Reply:
:mindblown: